x-pack/solutions/search/plugins/search_synonyms/public/components/synonym_sets/synonym_sets.tsx

```diff
@@ -44,7 +44,10 @@ export const SynonymSets = () => {
       }),
       render: (name: string) => (
         <div data-test-subj="synonyms-set-item-name">
-          <EuiLink onClick={() => application?.navigateToUrl(`${PLUGIN_ROUTE_ROOT}/sets/${name}`)}>
+          <EuiLink
+            data-test-subj="searchSynonymsColumnsLink"
+            onClick={() => application?.navigateToUrl(`${PLUGIN_ROUTE_ROOT}/sets/${name}`)}
```

Comment:
:x: these need `http.basePath.prepend` to navigate to the URL in hosted with or without a space defined.